libs/design-system/web-component-card/src/lib/web-component-card.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="image">
```

Comment:
There is no need to wrap the image element inside a div. You are setting the height of the image already so the extra div doesn't do anything anyways